data/caasp/update.sh

```diff
@@ -86,7 +86,7 @@ elif [ ! -z "${EXECUTE:-}" ]; then
     if [[ $EXECUTE == cmd.run* ]]; then
         $srun -P "$TARGET" cmd.run "${EXECUTE#* }"
     else
-        $srun -P "$TARGET" "$EXECUTE"
+        $srun -P "$TARGET" $EXECUTE
```

Comment:
you are very picky with the quotes here :)

Reply:
It's actually a fix:
```bash
# what we have now does not work, salt fails
salt '*' "grains.setval tx_update_reboot_needed True"

# after merge
salt '*' grains.setval tx_update_reboot_needed True
```

Reply:
Before: http://dhcp126.suse.cz/tests/14032#step/stack_update/7
After: http://dhcp126.suse.cz/tests/14038#step/stack_update/4

Reply:
ye it's bash, didn't notice, make sense